### DISCUSSION TITLE
search using m:

### DISCUSSION THREAD

**Discussion by dsm1212** [immich-app/immich#10275] [Q&A] [answered]
I used m: to search other metadata, but now I get fairly random results. I find nothing about it in the docs now so maybe it was removed?

Btw smart search seems to return some good results followed by total misses. I thought that previously I just got good results, but maybe I'm wrong. What I mean is, I search for bird. The first 100 photos are birds then I have a lot more photos shown with no birds.

**Comment by alextran1502** [comment]
Search using `m:` has been deprecated a while back. You can use the detail search panel to search for metadata instead

The search is ordered by the most relevant

**Comment by dsm1212** [comment, marked as answer]
Ok. Thanks!
